core/src/main/resources/reference.conf

```diff
@@ -53,17 +53,17 @@ kanaloa {
       # referenceDelay. E.g. if the estimated delay is 11 seconds, and the reference delay is 10 seconds,
       # the probability will increase by ((11 - 10) / 10) * 0.2 = 2%, this is will be applied every
       # updateInterval
-      delayFactorBase = 0.2
+      delayFactorBase = 0.05
```

Comment:
why were these base configs changed, out of curiosity?  If I understand, right, this PR was only affecting some metrics reporting and some of the stress testing code?